slices/tar.yaml

```diff
@@ -0,0 +1,31 @@
+package: tar
+
+essential:
+  - tar_copyright
+
+slices:
+  bins:
+    essential:
+      - tar_rmt
+      - tar_tar
+    contents:
+      /usr/sbin/tarcat:
```

Comment:
`tarcat` will actually need `dash` and `coreutils` (only `dd`, `tr`, `cut`).

Reply:
The `tar` package doesn't declare dependencies on `dash` & `coreutils`, so not sure I understand why the SDF would declare them?